autoPyTorch/pipeline/components/setup/network_backbone/ShapedResNetBackbone.py

```diff
@@ -38,10 +38,10 @@ def build_backbone(self, input_shape: Tuple[int, ...]) -> None:
         self.config.update(
             {"num_units_%d" % (i): num for i, num in enumerate(neuron_counts)}
         )
-        if self.config['use_dropout'] and self.config["max_dropout"] > 0.05:
+        if self.config['use_dropout']:
             dropout_shape = get_shaped_neuron_counts(
                 self.config['resnet_shape'], 0, 0, 1000, self.config['num_groups']
-            )
+            )[:-1]
```

Comment:
What is it supposed to mean?

Reply:
Since this was taken from my pr, to not write the same thing again, please take a look at https://github.com/automl/Auto-PyTorch/pull/245 at the first resolved comment.